pkg/cmd/dev/doctor.go

```diff
@@ -95,6 +96,30 @@ func (cfg *doctorConfig) maybePromptForAutofixPermission(question string) {
 	}
 }
 
+type buildConfig struct {
+	name        string
+	description string
+}
+
+// buildConfigs returns the available build configs (typically "dev" and
+// "crosslinux"). The first config will always be "dev".
+func buildConfigs() []buildConfig {
+	configs := []buildConfig{
+		{name: "dev", description: "uses the host toolchain"},
+	}
+	if runtime.GOOS == "linux" {
+		crossCfg := "crosslinux"
```

Comment:
(While it doesn't matter as the build is unlikely to work on these non-`amd64`, non-`arm64` systems, it's best if the code is written to be flexible rather than definitely broken.)

Reply:
TFTR! Good idea, done.